android/app/src/main/java/com/ptokenssentinelandroidapp/strongbox/Strongbox.java

```diff
@@ -345,7 +345,7 @@ private boolean verify(String alias, byte[] message, byte[] signature) {
         return false;
     }
 
-    private int getLatestAliasNumber() {
+    public static int getLatestAliasNumber() {
```

Comment:
Ah, so these were public for the state hash signing in the `DatabaseWiring` class.

I made them private when I had to expose an interface to this module to allow rust to be able to call stuff in it successfully.

Reply:
Ah I see, do you think they could break anything? They shouldn't in my opinion

Reply:
Nope - was just explaining why I changed them in the first place.